docker/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -xe
+#########Change ME###############
+export PYVER=2.7
```

Comment:
Any chance to parametrize this script in order not to have PYVER and CUDA hardcoded?

Reply:
Will do.